docs/getting-started/migrate-from-nexus-schema.md

```diff
@@ -137,6 +137,67 @@ You should only be working with the `nexus` CLI. Below shows the example scripts
 
 #### Other Details
 
+##### Backing Types
+
+With Nexus Schema backing types are handled as follows:
+
+- You co-locate TypeScript typings as strings with your object type configs
```

Comment:
Given that this is not even documented in the official doc, I probably wouldn't even mention it. WDYT? 🤔